Exercise_D and E/assets/style/index.css

```diff
@@ -1,4 +1,72 @@
-/* 
+/*
 This css sheet is linked to practice.html
 You can style tags, classes (.) or ids (#) from this file.
 */
+* {
+    margin: 0;
```

Comment:
Try to stick to one colour convention when coding, when you;re working in a company they will provide you with the palette and which type of colour to use.